### PR TITLE
Refs [TASK][27713] [trainer] Show course

### PR DESCRIPTION
**Pull request by ghost**
https://edu-redmine.sun-asterisk.vn/issues/27713
Screenshot from 2020-09-04 14-21-06

### CODE REVIEW

File: app/models/user_task.rb [hunk 1]
```diff
@@ -1,3 +1,6 @@
 class UserTask < ApplicationRecord
-  belongs_to :tasks
+  belongs_to :user_course_subject
+  belongs_to :task
+
+  scope :task_done, ->{where(status: 1).group(:user_course_subject_id).count}
```
Comment (manhnt-0960) [inline app/models/user_task.rb]: status: 1 e dùng enum method là dc, scope này e dùng group by thôi

File: app/views/partials/courses/edit/_subject.html.erb [hunk 2]
```diff
@@ -11,7 +11,8 @@
     <img class="img-thumbnail" id="img-preview" src="#" style="display: none" />
   <% end %>
   <div class="media-body">
-    <h5 class="mt-0"><%= item.name %></h5>
+    <h5 class="mt-0 font-weight-bold"><%= item.name %></h5>
+    <p class="text-success"><i class="fas fa-clock"></i> <%= item.duration %> days</p>
```
Comment (manhnt-0960) [inline app/views/partials/courses/edit/_subject.html.erb]: days

File: app/views/trainers/courses/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    <ul class="list-unstyled">
+      <%= render partial: "/partials/courses/trainee", collection: @users,
+                 as: :item %>
+    </ul>
+  </div>
+  <div class="tab-pane fade" id="contact" role="tabpanel" aria-labelledby="contact-tab">
+    <table class="table">
+      <thead>
+      <tr>
+        <th scope="col">Name</th>
```
Comment (manhnt-0960) [inline app/views/trainers/courses/show.html.erb]: Name

File: app/views/trainers/courses/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        <% @subjects.each do |subject| %>
+          <th scope="col">
+            <%= subject.name %>
+            <small class="text-muted">(Total task: <%= subject.tasks.size %>)</small>
```
Comment (manhnt-0960) [inline app/views/trainers/courses/show.html.erb]: i18n

File: app/views/partials/courses/edit/_subject.html.erb (hunk 2 above)
Comment (ghost) [reply app/views/partials/courses/edit/_subject.html.erb]: Vâng ạ

File: app/views/trainers/courses/show.html.erb (hunk 4 above)
Comment (ghost) [reply app/views/trainers/courses/show.html.erb]: Vaag a

**Comment by ghost** [comment]
ready

**Pull request merged**